### ISSUE TITLE
touchegg does not work with Ubuntu for ARM (Pi)

### ISSUE THREAD

**Issue by achixson** [JoseExposito/touchegg#275]
It does work very well with Mint.  I have a rather urgent need for touchegg on the Pi.  Any hope?

**Comment by JoseExposito** [comment]
I'm afraid you will need to compile some dependencies to get it running.

You can check the list of dependencies for your Ubuntu version here:
http://packages.ubuntu.com/search?keywords=touchegg&searchon=names

I don't know what packages are already in the Pi's repo, but I expect you to need to compile libgeis... And I don't expect it to be straightforward :S 

Good luck!

**Comment by achixson** [comment]
Thanks José,

The odd thing is that I'm running Ubuntu Mate that has been compiled by 
Ubuntu for the Pi.  Touchegg is in its app repository which means that 
the dependencies were also compiled for the ARM. So, compiling Touchegg 
probably will call for the same current dependencies.

I noticed that there were several other folks who updated their distros 
for whom Touchegg stopped working.  This suggests that there might have 
been some change in the X libraries that made it incompatible with Touchegg?

Everything I have compiled and installed has worked perfectly on Pi, 
including RealVNC.

For the application I'm working on I have a large touchscreen monitor.  
Pi uses HDMI or RCA for video output.  The unknowable (at this point) is 
whether Pi can receive touch screen output through HDMI.

Best wishes,
Arthur Hixson
